lms/services/dashboard.py

```diff
@@ -125,8 +168,11 @@ def get_request_admin_organizations(self, request) -> list[Organization]:
                 )
             ).all()
 
-        return self.get_organizations_by_admin_email(
-            request.lti_user.email if request.lti_user else request.identity.userid
+        return self.get_organizations_where_admin(
+            h_userid=request.lti_user.h_user.userid(self._h_authority),
+            email=request.lti_user.email
+            if request.lti_user
+            else request.identity.userid,
```

Comment:
Is `identity.userid` an email? 🤔

Reply:
Yes, but only for staff, their gmail address.